hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/client/HoodieFlinkWriteClient.java

```diff
@@ -281,7 +281,11 @@ public void initMetadataWriter() {
    * checkpoint finish.
    */
   public void startAsyncCleaning() {
-    this.asyncCleanerService = AsyncCleanerService.startAsyncCleaningIfEnabled(this);
+    if (this.asyncCleanerService == null) {
```

Comment:
some way to simplify this if-else into a single line. Can we use option instead of null?

Reply:
Option is designed to be used a method return value not class member.